docs/platform/concepts/service_backups.rst

```diff
@@ -166,11 +170,12 @@ Aiven for ClickHouse backups contain database lists, table schemas, table conten
 Access to backups
 -----------------
 
-The Aiven platform provides a centralised managed platform for Aiven services to run across many different cloud providers and regions. Tooling that built to provide the service backups are open source and available for you to use in your own infrastructure. 
+The Aiven platform provides a centralized managed platform for Aiven services, enabling them to run on various cloud providers and regions. The open-source tooling provided for service backups is also available for use in your own infrastructure.
+
+The Aiven platform is designed to handle the operational aspects of running complex software at scale, allowing you to focus on using the services instead of maintaining them. Aiven handles service availability, security, connectivity, and backups.
 
-The nature of the Aiven platform is to manage the operational tasks of running complex software at scale so that you are able to focus your efforts on using the services, not maintaining them. Aiven takes care of service availability, security, connectivity, and backups.
+Access to backups of your services is not possible due to their encryption and storage in object storage. However, if you do need to backup your service, you can use the standard tooling for this service.
```

Comment:
Consider: I'd put justification/reasoning first.

Since service backups are encrypted and stored in the object storage, accessing them is not possible. If you do need to backup your service, use the standard tooling for this service.